### PR TITLE
refactor menu

### PR DESCRIPTION
**Pull request by karissa**
:file_cabinet: :factory: :recycle: :relaxed: cleaning house

still didn't find a broom emoji though, what's with that?

### CODE REVIEW

File: src/renderer/components/Menu.js [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+const dialogs = require('./dialogs')
+const {
+  Menu,
+  MenuItem
+} = require('@blueprintjs/core')
+
+class Controller {
+  constructor (props) {
+    this.props = props
+    autobind(this)
```
Comment (alfaslash) [inline src/renderer/components/Menu.js]: A good solution.
But how about use [React-autobind](https://github.com/cassiozen/React-autobind) instead [class-autobind](https://github.com/kodefox/class-autobind)?

`react-autobind` allow specify the only methods that should be auto-bound. And in other components  you can use auto-bound without create new class (ex. `Controller`).

Comment (ralphtheninja) [reply src/renderer/components/Menu.js]: @alfaslash @karissa is on vacation right now and we want to move as fast as possible atm, so things aren't perfect. If you think this important, make an issue and link to this PR _or_ make a PR :)

Comment (karissa) [reply src/renderer/components/Menu.js]: @alfaslash in this case we wanted all of the methods to be bound to the controller so this solution seemed to work well.

Comment (karissa) [reply src/renderer/components/Menu.js]: class-autobind uses decorators which requires more babel transforms

**Pull request merged**